learn/sherif/main_test/src/main/java/frc/robot/subsystems/ShooterController.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.helpers.MockableSparkMax;
+import frc.robot.interfaces.CANSparkMaxInterface;
+
+public class ShooterController extends SubsystemBase {
+    private final CANSparkMaxInterface leftSide;
+    private final CANSparkMaxInterface rightSide;
+    private final boolean useFollower;
+
+    public double getSpeed() {
+        return speed;
+    }
+
+    public void setSpeed(double speed) {
+        this.speed = speed;
+    }
+
+    private double speed = 1.0;
```

Comment:
Maybe add the default speed to Constants.java?

Reply:
Also, I would suggest you make this 0 rather than 1, bc we don't want the shooter wheel to start the match spinning at max speed